### ISSUE TITLE
artifact on ribbon plots on GR

### ISSUE THREAD

**Issue by piever** [JuliaPlots/Plots.jl#1123]
```julia
julia> using Plots
julia> plot(sin, ribbon = 0.1)
```
![gr_ribbon](https://user-images.githubusercontent.com/6333339/31075963-f4d8681a-a76f-11e7-8288-ef0fcd6dd60d.png)

gives tiny white lines in the "ribbon" part. At some point it also happened that the plot line was "broken" but that was harder to reproduce. It seems that it's plotting every little chunk individually and they are not properly connected.

Here is a zoomed version:

![screen shot 2017-10-02 at 12 52 03](https://user-images.githubusercontent.com/6333339/31076084-9203b2de-a770-11e7-8110-cc656ec609b1.png)

I'm on Plots master and see this behavior both on GR release and master.

**Comment by daschw** [comment]
That's probably related to the automatic choices of the x values:
```julia
x = linspace(-5, 5, 10000)
plot(x, sin.(x), ribbon = 0.1)
```
looks better.

**Comment by SimonDanisch** [comment]
i'm pretty sure that this is a rendering artifact... Choosing different defaults just seems to hide it ;)

**Comment by piever** [comment]
I agree with @SimonDanisch : we can't expect the user to hide artefacts by choosing 10000 points on the x axis... Maybe it's better to ping @jheinen on this one, as I really don't know much about the details of the GR rendering system.

**Comment by daschw** [comment]
ok

**Comment by jheinen** [comment]
I have to check how ribbon plots are implemented (emulated) in Plots' GR backend ...

**Comment by daschw** [comment]
@jheinen, I think, they are just fillranges, set for all backends in https://github.com/JuliaPlots/Plots.jl/blob/master/src/utils.jl#L483-L500

**Comment by daschw** [comment]
@piever solved the issue! 🎉
It was caused by myself in #1110 (cf. #1139) and will be fixed by #1142.
